lib/cli-engine.js

```diff
@@ -402,6 +457,12 @@ CLIEngine.prototype = {
 
         stats = calculateStatsPerRun(results);
 
+        if (options.useCache) {
+            // persist the cache to disk
+            fCache.reconcile();
+        }
+
+        debug("Time: " + (Date.now() - startTime) + "ms");
```

Comment:
I don't like these implicit conversions. Use `getTime` or unary `+` to explicitly convert dates to numbers.